pythran/tests/test_numpy_func0.py

```diff
@@ -370,6 +372,48 @@ def test_identity0(self):
     def test_identity1(self):
         self.run_test("def np_identity1(a): from numpy import identity ;return identity(a)", 4, np_identity1=[int])
 
+    def test_fromfile0(self):
+        temp_name = tempfile.mkstemp()[1]
+        x = numpy.random.randint(0,2**8,1000).astype(numpy.uint8)
+        x.tofile(temp_name)
```

Comment:
in case of exception, the created file will not get removed. Please use an exception handler to ensure the file removal.